webapp/babel.config.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const config = {
+    presets: [
+        ['@babel/preset-env', {
+            targets: {
+                chrome: 66,
+                firefox: 60,
+                edge: 42,
+                safari: 12,
```

Comment:
have we tested the UI flow on these browsers?

Reply:
Yes because its explicitly mentioned to support these browsers so it would be good to do a test.
You can use [LamdaTest](https://app.lambdatest.com/console/realtime/browser?_gl=1*1ty4cl*_gcl_aw*R0NMLjE2OTUxMTI3MDAuQ2p3S0NBandqYVdvQmhBbUVpd0FYejhEQldDbnhyV3F5TngyTDZYbXc2VkdpV2ZZbzlZQjVNTUdSc21GQTJvcXpoNndLRkpTMDVkczJCb0MtQXdRQXZEX0J3RQ..*_gcl_au*MjA3ODUxNjYxLjE2OTUxMTI2OTk.) with BS email ID to test on Safari and Edge.

Reply:
As discussed with you, BS email doesn't have a subscription to LambaTest now. I conveyed the message to Kshitij, and he will inform this to Mick

Reply:
Make sure to follow up on this.